src/components/Content/Content.js

```diff
@@ -69,12 +119,35 @@ function Content(props) {
                 </AwesomeButtonSocial>
               </Grid>
             </Grid>
+
           </div>
         </Container>
+        <Container maxWidth='md'>
+          <Grid container spacing={1} justify='center'>
+            <Grid item xs={12} sm={8} md={4}>
+              <SearchTextField
+                InputLabelProps={{
+                  className: classes.floatingLabelFocusStyle,
+                }}
+                onChange={handleSearch}
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                style={{ marginTop: 40 }}
+                fullWidth
+                margin="normal"
+                variant="outlined"
+                placeholder="Search by name or Github username"
+                id="input-with-icon-grid"
+                color="secondary"
+                label="🔎" />
```

Comment:
Remove this label. It will look better without it. Need to work on styling the Search field.

Reply:
Removed `label`.
I would suggest to raise a new issue for styling `Search field`, so that someone who is interested in styling make use of this!

Reply:
Sure, Got your point! Styles now look a lot better! Let's just add '🔍' icon to the search field for now. That's the only thing which I think needed for now to close this issue.